docs/rest_api.rst

```diff
@@ -640,6 +640,47 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+Querying connections details
+------------------------------
+
+You can query for details of previously joined token networks by making a GET request to the connection endpoint.
+
+``GET /api/<version>/connection``
```

Comment:
please rename the endpoint here and elsewhere in the code to its plural form: `connections`

Reply:
Wait no. Disregard this comment, we already use this.